### ISSUE TITLE
AbjadBook: Fix print(x, y, z) output.

### ISSUE THREAD

**Issue by josiah-wolf-oberholtzer** [Abjad/abjad#501]
The following .rst input...

```
..  abjad::

    staff = Staff("c'4 d'4 e'4 f'4")
    spanner = Slur()
    attach(spanner, staff[:])

..  abjad::

    for leaf in staff:
        is_first = spanner._is_my_first_leaf(leaf)
        is_last = spanner._is_my_last_leaf(leaf)
        print(leaf, is_first, is_last)
```

...produces the following doctree output:

```
<document source="test">
    <block_quote>
        <literal_block xml:space="preserve">
            >>> staff = Staff("c'4 d'4 e'4 f'4")
            >>> spanner = Slur()
            >>> attach(spanner, staff[:])
        <literal_block xml:space="preserve">
            >>> for leaf in staff:
            ...     is_first = spanner._is_my_first_leaf(leaf)
            ...     is_last = spanner._is_my_last_leaf(leaf)
            ...     print(leaf, is_first, is_last)
            ...
            c'4

            True

            False
            d'4

            False

            False
            e'4

            False

            False
            f'4

            False

            True
```

... and not:

```
<document source="test">
    <block_quote>
        <literal_block xml:space="preserve">
            >>> staff = Staff("c'4 d'4 e'4 f'4")
            >>> spanner = Slur()
            >>> attach(spanner, staff[:])
        <literal_block xml:space="preserve">
            >>> for leaf in staff:
            ...     is_first = spanner._is_my_first_leaf(leaf)
            ...     is_last = spanner._is_my_last_leaf(leaf)
            ...     print(leaf, is_first, is_last)
            ...
            c'4 True False
            d'4 False False
            e'4 False False
            f'4  False True
```

**Comment by josiah-wolf-oberholtzer** [comment]
It may be necessary to proxy the builtin `print()` function.

**Comment by josiah-wolf-oberholtzer** [comment]
Fixed in 0b11a4b193072a084a8f64dad3c030647066b2bf.
